mis_builder_total_committed_purchase/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+The mis_builder `roadmap <https://github.com/OCA/mis-builder/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement>`_
+and `known issues <https://github.com/OCA/mis-builder/issues?q=is%3Aopen+is%3Aissue+label%3Abug>`_ can
+be found on github.
```

Comment:
```suggestion
The mis_builder `roadmap <https://github.com/OCA/mis-builder-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement>`_
and `known issues <https://github.com/OCA/mis-builder-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Abug>`_ can
be found on github.
```
Maybe it should now be relative to mis-builder-contrib repo ? Since it has been moved